README.md

```diff
@@ -1,117 +1,108 @@
 # OpenZeppelin Contracts for Cairo
 
-[![Tests and linter](https://github.com/OpenZeppelin/cairo-contracts/actions/workflows/coverage.yml/badge.svg)](https://github.com/OpenZeppelin/cairo-contracts/actions/workflows/coverage.yml)
-[![codecov](https://codecov.io/github/OpenZeppelin/cairo-contracts/branch/main/graph/badge.svg?token=LFSZH8RPOL)](https://codecov.io/github/OpenZeppelin/cairo-contracts)
+[![Lint and test](https://github.com/OpenZeppelin/cairo-contracts/actions/workflows/test.yml/badge.svg)](https://github.com/OpenZeppelin/cairo-contracts/actions/workflows/test.yml)
 
-**A library for secure smart contract development** written in Cairo for [StarkNet](https://starkware.co/product/starknet/), a decentralized ZK Rollup.
+**A library for secure smart contract development** written in Cairo for [Starknet](https://starkware.co/product/starknet/), a decentralized ZK Rollup.
 
-## Usage
-
-> ## ⚠️ WARNING! ⚠️
->
+> **Warning**
 > This repo contains highly experimental code.
-> Expect rapid iteration.
+> It has no code coverage checks.
+> It hasn't been audited.
 > **Use at your own risk.**
 
-### First time?
+## Usage
 
-Before installing Cairo on your machine, you need to install `gmp`:
+> **Warning**
+> Expect rapid iteration.
+> Some contracts or features are not ready to be deployed.
+> Check the **Unsupported** section below.
 
-```bash
-sudo apt install -y libgmp3-dev # linux
-brew install gmp # mac
-```
+### Prepare the environment
 
-> If you have any troubles installing gmp on your Apple M1 computer, [here’s a list of potential solutions](https://github.com/OpenZeppelin/nile/issues/22).
+1. [Install cairo](https://book.cairo-lang.org/ch01-01-installation.html)
+2. [Install scarb](https://docs.swmansion.com/scarb/download)
 
 ### Set up your project
 
-Create a directory for your project, then `cd` into it and create a Python virtual environment.
+Create a new project and `cd` into it.
 
 ```bash
-mkdir my-project
-cd my-project
-python3 -m venv env
-source env/bin/activate
+scarb new my_project && cd my_project
```

Comment:
This line doesn't start with `$ ` while the next one does. I think we should keep the consistency either way.

Reply:
the criteria is for `$` to be there only when the output is also present. when no output is shown, the $ is not needed (preferable for copypasting).

Reply:
I would rather stick with `$`, but just because it looks better to me :), there is no strong defense besides that.

Reply:
Well, I saw now the Markdown Linter not accepting dollar signs when there is no output :). Is ok to leave it as it was then.